### PR TITLE
Translate to Indonesia

### PR DESCRIPTION
**Pull request by agungsetiawan**

### CODE REVIEW

**Comment by waruboy** [comment]
Thanks bro @agungsetiawan . Ng, instead of replacing, how about just creating a new file, like coc-id.md. Sorry if I am not clear about this before

File: conf/2017/coc.md [hunk 1]
```diff
@@ -3,30 +3,30 @@ layout: conf
...
+<p>Konferensi kita ini didedikasikan untuk penyelenggaraan konferensi yang bebas pelecehan untuk semua orang, tidak memandang jenis kelamin, identitas gender dan ekspresi gender, orientasi seksual, disabilitas, bentuk fisik, ukuran tubuh, ras, etnis, agama (atau kepercayaan), atau pilihan teknologi. Kami tidak menoleransi tidak pelecehan terhadap peserta konferensi dalam bentuk apapun. Bahasa seksual dan perumpamaannya tidak diperbolehkan untuk venue apa pun, termasuk acara diskusi, workshop, pesta, Twitter dan media online lainnya. Peserta konferensi yang melanggar aturan-aturan ini akan diberi sanksi atau diusir dari konferensi <em>tanpa pengembalian dana</em> sesuai kebijaksanaan penyelenggara.</p>
```
Comment (waruboy) [inline conf/2017/coc.md]: tidak pelecehan -> tindak pelecehan

Comment (waruboy) [inline conf/2017/coc.md]: missed: "umur"

File: conf/2017/coc.md [hunk 2]
```diff
@@ -3,30 +3,30 @@ layout: conf
...
+<p>Peserta yang diminta untuk menghentikan tindakan pelecehannya diharapkan segera mematuhi.</p>
 
-<p>Sponsors are also subject to the anti-harassment policy. In particular, sponsors should not use sexualized images, activities, or other material. Booth staff (including volunteers) should not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.</p>
+<p>Sponsor juga dikenakan aturan anti-pelecehan. Terkhusus untuk sponsor, tidak diperbolehkan menggunakan gambar, aktifitas, atau materi lain yang bersifat seksual. Staf stan (termasuk sukarelawan) tidak diperbolehkan menggunakan pakaian/seragam/kostum bersifat seksual, atau membuat suasana seksual.</p>
 
-<p>If a participant engages in harassing behavior, the conference organisers may take any action they deem appropriate, including warning the offender or expulsion from the conference with no refund.</p>
+<p>Jika peserta terlibat pelecehan, penyelenggara konferensi akan mengambil langkah yang dianggap perlu, termasuk memberikan peringatan kepada penyerang atau diusir dari konferensi tanpa pengembalian dana.</p>
```
Comment (waruboy) [inline conf/2017/coc.md]: penyerang > pelaku pelecehan

File: conf/2017/coc.md [hunk 3]
```diff
@@ -3,30 +3,30 @@ layout: conf
...
+<p>Jika kamu dilecehkan, melihat orang lain dilecehkan, atau punya kepentingan lainnya, silakan hubungi staf penyelenggara segera. Panitia dapat dikenali dengan kaos khusus panitia yang digunakan.</p>
 
-<p>Conference staff will be happy to help participants contact hotel/venue security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.</p>
+<p>Panitia bersedia dengan senang hati membantu peserta untuk menghubungi petugas keamanan gedung/tempat acara atau penegak hukum, menyediakan pengawalan, atau bantuan lain untuk mereka yang mengalami pelecehan supaya merasa aman selama konferensi berlangsung. Kami menghargai kedatanganmu.</p>
 
-<p>We expect participants to follow these rules at conference and workshop venues and conference-related social events.</p>
\ No newline at end of file
+<p>Kami berharap panitia untuk mengikuti aturan-aturan tersebut di konferensi ini dan di tempat workshopo dan acara sosial lain yang berkaitan dengan konferensi ini.</p>
```
Comment (waruboy) [inline conf/2017/coc.md]: panitia > peserta

**Comment by waruboy** [comment]
LGTM

**Comment by firewalker06** [comment]
Sorry, the RubyConf page layout has changed into single page layout. Could you remake in another PR based on the current layout? 🙇

**Pull request closed without merging**